Formula/red.rb

```diff
@@ -0,0 +1,16 @@
+class Red < Formula
+  desc "Next-gen programming language, strongly inspired by REBOL"
+  homepage "http://www.red-lang.org"
+  url "http://static.red-lang.org/dl/mac/red-061"
```

Comment:
In that case this doesn't look like it's open-source so is not a good fit for Homebrew. Homebrew Cask may accept it.